Changes.md

```diff
@@ -1,5 +1,15 @@
 # Medic Mobile Release Notes
 
+## 2.8.6
+
+### TBD
+
+#### Continuous Integration
+
+- Automatically update to master branches of `api` and `sentinel` submodules
+  before running CI in Travis. [#3621](https://github.com/medic/medic-webapp/pull/3621)
+- Fixed kanso packages that inadvertantly cached data. Issue: #3648
+
```

Comment:
The first item isn't included in this release - remove it.

The second item isn't a CI issue, it's a security one, so change the header to Security.